go/common/sgx/pcs/tcb.go

```diff
@@ -271,15 +272,21 @@ func (ti *TCBInfo) validateTCBLevel(
 	switch tcbLevel.Status {
 	case StatusUpToDate, StatusSWHardeningNeeded:
 		// These are ok.
-	default:
-		return &TCBOutOfDateError{
-			Kind:        TCBKindPlatform,
-			Status:      tcbLevel.Status,
-			AdvisoryIDs: tcbLevel.AdvisoryIDs,
+		return nil
+	case StatusOutOfDate, StatusOutOfDateConfigurationNeeded:
+		// Ok if `OASIS_UNSAFE_LAX_AVR_VERIFY` is set.
+		if os.Getenv("OASIS_UNSAFE_LAX_AVR_VERIFY") != "" {
```

Comment:
Don't do it like that as it seems quite dangerous. I would handle it similarly to how we have `unsafeAllowDebugEnclaves` and then set this in `oasis-node/cmd/common` based on some debug configuration option (gated also on `flags.DebugDontBlameOasis()`).

Reply:
Updated